### PR TITLE
Latest vmmlib API

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by bilgili** [comment]
I understand these are good changes for unifying the naming but can not we do it after the release ?

**Comment by eile** [comment]
Yes, see comment in Eyescale/vmmlib#50

**Comment by eile** [comment]
ping

**Comment by eile** [comment]
ping?

**Comment by eile** [comment]
Retest this please.

File: apps/livreGUI/CMakeLists.txt [hunk 1]
```diff
@@ -61,4 +61,4 @@ list(APPEND CPPCHECK_EXTRA_ARGS
 
 add_definitions(${QT_DEFINITIONS} "-DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED" )
 
-common_gui_application(livreGUI)
+common_application(livreGUI GUI)
```
Comment (bilgili) [inline apps/livreGUI/CMakeLists.txt]: GUI is a keyword ?

File: livre/eq/Channel.cpp [hunk 2]
```diff
@@ -533,10 +533,8 @@ struct Channel::Impl
         LBASSERT( !_channel->useOrtho( ));
 
         // calculate modelview inversed+transposed matrix
-        Matrix3f modelviewITM;
-        Matrix4f modelviewIM;
-        modelView.inverse( modelviewIM );
-        Matrix3f( modelviewIM ).transpose_to( modelviewITM );
+        const Matrix4f& modelviewIM = modelView.inverse();
+        const Matrix3f& modelviewITM = vmml::transpose( Matrix3f( modelviewIM ));
```
Comment (bilgili) [inline livre/eq/Channel.cpp]: Matrix3f is automatically having the upper-left corner of Matrix4f ?

File: livre/eq/Channel.cpp [hunk 3]
```diff
@@ -533,10 +533,8 @@ struct Channel::Impl
...
 
         Vector3f norm = modelviewITM * Vector3f( 0.0f, 0.0f, 1.0f );
```
Comment (bilgili) [inline livre/eq/Channel.cpp]: there is no free normalize function ?

File: apps/livreGUI/CMakeLists.txt (hunk 1 above)
Comment (eile) [inline apps/livreGUI/CMakeLists.txt]: An argument: https://github.com/Eyescale/CMake/blob/master/CommonApplication.cmake

File: livre/eq/Channel.cpp (hunk 2 above)
Comment (eile) [inline livre/eq/Channel.cpp]: https://github.com/Eyescale/vmmlib/blob/master/vmmlib/matrix.hpp#L72

File: livre/eq/Channel.cpp (hunk 3 above)
Comment (eile) [inline livre/eq/Channel.cpp]: https://github.com/Eyescale/vmmlib/blob/master/vmmlib/vector.hpp#L428

**Comment by bilgili** [comment]
+1

**Pull request merged**